src/scss/grommet-core/_objects.accordion-panel.scss

```diff
@@ -0,0 +1,18 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+.grommet .#{$grommet-namespace}accordion-panel {
+
+  > .#{$grommet-namespace}box--clickable {
```

Comment:
Can we give the Box a class name specific to the accordion context to avoid the direct child selector and keep the CSS flatter?